tests/client-sdk/memory/resources/run.yaml

```diff
@@ -0,0 +1,108 @@
+version: '2'
```

Comment:
Remove? This is a duplicate of https://github.com/meta-llama/llama-stack/blob/main/llama_stack/templates/ollama/run.yaml?

We should be able to run tests on all template/*/run.yaml configs.